File: 2023/php/test/TestDay2B.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day2\b\Solution2B;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay2B extends TestCase
{
    public function testExample(): void
    {
        $this->assertSame(2286, Solution2B::getResult("../resources/2/test_b.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(68638, Solution2B::getResult("../resources/2/input.txt", TestLogger::createTestLogger()));
    }

}

File: 2023/java/src/test/java/org/frieder/aoc/day2/TestDay2B.java
package org.frieder.aoc.day2;

import static org.junit.Assert.assertEquals;

import org.frieder.aoc.day2.b.Solution2B;
import org.junit.Test;

import java.io.IOException;

public class TestDay2B {
    @Test
    public void testExample() throws IOException {
        assertEquals(2286, Solution2B.getResult("../resources/2/test_b.txt"));
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(68638, Solution2B.getResult("../resources/2/input.txt"));
    }
}
